packages/bff/src/model/domain/errors.ts

```diff
@@ -23,17 +23,20 @@ export const errorCodes = {
   invalidRiskAnalysisContentType: "0015",
   missingInterface: "0016",
   eserviceRiskNotFound: "0017",
+  invalidInterfaceContentTypeDetected: "0018",
+  invalidInterfaceFileDetected: "0019",
+  openapiVersionNotRecognized: "0020",
+  interfaceExtractingInfoError: "0021",
+  agreementDescriptorNotFound: "0022",
+  contractNotFound: "0023",
+  contractException: "0024",
+  invalidContentType: "0025",
   noDescriptorInEservice: "0018",
   missingDescriptorInClonedEservice: "0019",
-  invalidInterfaceContentTypeDetected: "0020",
-  invalidInterfaceFileDetected: "0021",
-  openapiVersionNotRecognized: "0022",
-  interfaceExtractingInfoError: "0023",
-  agreementDescriptorNotFound: "0024",
-  unknownTenantOrigin: "0025",
-  invalidJwtClaim: "0026",
-  samlNotValid: "0027",
-  missingSelfcareId: "0028",
+  unknownTenantOrigin: "0020",
+  invalidJwtClaim: "0021",
+  samlNotValid: "0022",
+  missingSelfcareId: "0023",
```

Comment:
Same comment left in the other PR 😬 Could you add all and only the new errors as the last ones, without changing errors already defined?